src/Cooltip/LabelComponent.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import styled from 'styled-components';
+import * as styles from '../Input/styles';
+
+const LabelComponent = styled.label`
+  font-size: ${styles.fontSize};
+  font-weight: ${styles.fontWeight};
+  display: block;
+`;
+
+export default LabelComponent;
```

Comment:
Can you just use this guy? https://github.com/mango-chutney/mango-components/blob/67cf963c0fe98b179f2ca448bf8f63a4de028367/src/Input/InputLabelComponent.js